examples/kubernetes/connectivity-check/Makefile

```diff
@@ -1,35 +1,90 @@
 # Copyright 2017-2020 Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
-#
+
+include ../../../Makefile.defs
+include ../../../Makefile.quiet
+
 DEFAULT_OUT := connectivity-check.yaml
 HOSTPORT_OUT := connectivity-check-hostport.yaml
 SINGLE_OUT := connectivity-check-single-node.yaml
+PROXY_OUT := connectivity-check-proxy.yaml
+QUARANTINE_OUT := connectivity-check-quarantine.yaml
+
+SERVERS_NAME := echo-a echo-b echo-b-host echo-b-host-headless echo-c echo-c-host echo-c-host-headless
+SERVERS_OUT := servers.yaml
 
-SRC := $(wildcard *.yaml)
-HOSTPORT_SRC := $(filter-out $(DEFAULT_OUT) $(HOSTPORT_OUT) $(SINGLE_OUT), $(SRC))
-DEFAULT_SRC := $(filter-out $(wildcard *-hostport.yaml),$(HOSTPORT_SRC))
-SINGLE_SRC := $(filter-out $(wildcard *-multi-node*.yaml),$(DEFAULT_SRC))
+SRC := $(wildcard *.cue)
+ALL_TARGETS := $(DEFAULT_OUT) $(HOSTPORT_OUT) $(SINGLE_OUT) $(PROXY_OUT) $(QUARANTINE_OUT)
 
+DOCKER_RUN := $(CONTAINER_ENGINE) container run --rm \
+	--workdir /src/examples/kubernetes/connectivity-check \
+	--volume $(CURDIR)/../../..:/src \
+	--user "$(shell id -u):$(shell id -g)"
 
-all: $(DEFAULT_OUT) $(HOSTPORT_OUT) $(SINGLE_OUT)
+CUE_IMAGE := "docker.io/cuelang/cue:v0.2.2@sha256:2ea932c771212db140c9996c6fff1d236f3f84ae82add914374cee553b6fc60c"
+CUE := $(DOCKER_RUN) $(CUE_IMAGE)
 
-$(DEFAULT_OUT): $(DEFAULT_SRC)
+all: $(ALL_TARGETS)
```

Comment:
nit: might need to add step to run this target and check `git diff` before running smoke test. Can be done as follow up.

Reply:
Filed https://github.com/cilium/cilium/issues/12714.